util/network/cniprovider/createns_linux.go

```diff
@@ -91,8 +91,8 @@ func unmountNetNS(nativeID string) error {
 }
 
 func deleteNetNS(nativeID string) error {
-	if err := os.RemoveAll(filepath.Dir(nativeID)); err != nil && !errors.Is(err, os.ErrNotExist) {
-		return errors.Wrap(err, "error removing network namespace")
+	if err := os.RemoveAll(nativeID); err != nil && !errors.Is(err, os.ErrNotExist) {
```

Comment:
`os.Remove`, also rename `nativeID` variable as it is not ID

Reply:
can remove the `deleteNetNS` call in line 20

Reply:
Done & done.